app/features/accounts/account.ts

```diff
@@ -97,3 +103,27 @@ export const getAccountBalance = (account: Account) => {
   }
   return account.balance;
 };
+
+/**
+ * Compares two account objects to determine if they are effectively the same account.
+ */
+export const isSameEffectiveAccount = (a: Account, b: Account): boolean => {
```

Comment:
I don't like this name. I would just call it isSameAccount maybe

Reply:
I removed this and created new `isClaimingToSameCashuAccount` method which makes it less confusing imo. Because for spark concept of account being same doesn't even exist